<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>832</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #282828;
color: #969696;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Rectangle4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>268</width>
      <height>832</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>832</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="MaterialVis">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>31</y>
       <width>183</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* MaterialVis */

font-family: 'Inter';
font-style: normal;
font-weight: 800;
font-size: 32px;
line-height: 39px;
/* identical to box height */


color: #00897B;
</string>
     </property>
     <property name="text">
      <string>Material&lt;font color='#00564D'&gt;Vis&lt;/font&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="date" native="true">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>104</y>
       <width>176</width>
       <height>172</height>
      </rect>
     </property>
     <widget class="QWidget" name="Size" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>166</width>
        <height>63</height>
       </rect>
      </property>
      <widget class="QLabel" name="Cube_size">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>103</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Cube size: */


position: absolute;
width: 103px;
height: 24px;
left: 49px;
top: 104px;

font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 20px;
line-height: 24px;

color: #969696;</string>
       </property>
       <property name="text">
        <string>Cube size:</string>
       </property>
       <property name="buddy">
        <cstring>Rectangle8</cstring>
       </property>
      </widget>
      <widget class="QWidget" name="Rectangle8_Wid" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>42</y>
         <width>166</width>
         <height>21</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="Rectangle8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>166</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Rectangle 8 */


background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="Points" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>109</y>
        <width>176</width>
        <height>63</height>
       </rect>
      </property>
      <widget class="QLabel" name="Number_of_points">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>176</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Number of points: */


position: absolute;
width: 176px;
height: 24px;
left: 48px;
top: 213px;

font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 20px;
line-height: 24px;

color: #969696;</string>
       </property>
       <property name="text">
        <string>Number of points:</string>
       </property>
       <property name="buddy">
        <cstring>Rectangle9</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="Rectangle9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>42</y>
         <width>166</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Algorithm" native="true">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>338</y>
       <width>168</width>
       <height>298</height>
      </rect>
     </property>
     <widget class="QPushButton" name="Algorithm1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>165</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Rectangle 3 */
QPushButton {
background: #282828;
border: 1px solid #000000;
border-radius: 15px;
font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;
color: #171616;}

QGraphicsDropShadowEffect {
blurRadius: 5px;
color: rgba(0, 0, 0, 0.25);
xOffset: 4px;
yOffset: 4px;
}

</string>
      </property>
      <property name="text">
       <string>Algorithm 1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Algorithm2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>87</y>
        <width>165</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

background: #282828;
border: 1px solid #000000;
border-radius: 15px;
font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;

color: #171616;</string>
      </property>
      <property name="text">
       <string>Algorithm 2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Algorithm3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>174</y>
        <width>165</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

background: #282828;
border: 1px solid #000000;
border-radius: 15px;
font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;

color: #171616;</string>
      </property>
      <property name="text">
       <string>Algorithm 3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Algorithm4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>261</y>
        <width>165</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background: #282828;
border: 1px solid #000000;
border-radius: 15px;

font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;

color: #171616;</string>
      </property>
      <property name="text">
       <string>Algorithm 4</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>Algorithm2</zorder>
     <zorder>Algorithm3</zorder>
     <zorder>Algorithm4</zorder>
     <zorder>Algorithm1</zorder>
    </widget>
    <widget class="QPushButton" name="Save">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>746</y>
       <width>165</width>
       <height>37</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #282828;
border: 1px solid #5C5C5C;
border-radius: 15px;
font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;

color: #5C5C5C;</string>
     </property>
     <property name="text">
      <string>Save project</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Tab">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>0</y>
      <width>1012</width>
      <height>832</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
padding : 0 ;
margin: 0;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="Rectangle1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>41</y>
       <width>1012</width>
       <height>791</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Rectangle 1 */


position: absolute;
width: 1012px;
height: 791px;
left: 268px;
top: 41px;

background: #363636;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="Visibility" native="true">
      <property name="geometry">
       <rect>
        <x>802</x>
        <y>37</y>
        <width>167</width>
        <height>45</height>
       </rect>
      </property>
      <widget class="QSlider" name="Rectangle10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>167</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 86, 77, 0.5) , stop:1 rgba(0, 137, 123, 0.5));
    height: 14px;
    margin: -1px 0;
    border-radius: 7px;
 }

 QSlider::handle:horizontal {
   position: absolute;
   background: #969696;
    width: 16px;
    height: 16px;
    margin: -1px 0;
    padding: 0;
    border-radius: 8px;
    subcontrol-position: center;
    subcontrol-origin: margin;
 }
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="Exploded_view">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>27</y>
         <width>114</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;

color: #969696;</string>
       </property>
       <property name="text">
        <string>Exploded view</string>
       </property>
       <property name="buddy">
        <cstring>Rectangle10</cstring>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wid_color" native="true">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>727</y>
        <width>127</width>
        <height>19</height>
       </rect>
      </property>
      <widget class="QCheckBox" name="Colormap">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>127</width>
         <height>19</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
QCheckBox {
    font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 19px;
padding-left: 0;
color: #969696;
}

QCheckBox::indicator {

width: 16px;
height: 16px;

background: #363636;
border: 1px solid #969696;
border-radius: 2px;
}

QCheckBox::indicator:checked {
background-color: #969696;
border: 1px solid #969696;
}</string>
       </property>
       <property name="text">
        <string>Choose color</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wid_start" native="true">
      <property name="geometry">
       <rect>
        <x>689</x>
        <y>644</y>
        <width>280</width>
        <height>93</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Start">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>280</width>
         <height>93</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #282828;
border-radius: 8px;

font-family: 'Inter';
font-style: normal;
font-weight: 700;
font-size: 48px;
line-height: 58px;
/* identical to box height */


color: rgba(150, 150, 150, 0.5);
padding: 18px 61px 17px 60px;</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1012</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #282828;</string>
     </property>
    </widget>
   </widget>
   <zorder>Tab</zorder>
   <zorder>Rectangle4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
